src/lib/operators/get_table.cpp

```diff
@@ -0,0 +1,18 @@
+#include "get_table.hpp"
+#include "storage/storage_manager.hpp"
+
+namespace opossum {
+
+GetTable::GetTable(const std::string& name) {
+  _table_name = name;
+}
```

Comment:
```suggestion
GetTable::GetTable(const std::string& name) : _table_name{name} {}
```
Maybe use the member initializer list to avoid assignments in the constructor body.